sinicum-server/src/main/java/com/dievision/sinicum/server/jcr/WysiwygTemplateTranslator.java

```diff
@@ -107,4 +109,21 @@ private String getFingerprint(Node node, Node jcrContent) throws RepositoryExcep
         }
         return fingerprint;
     }
+
+    private String multisiteAwarePath(String path) throws RepositoryException {
+        NodeIterator it = getMultisiteNodeIterator();
+        while (it.hasNext()) {
+            Node multisiteNode = it.nextNode();
```

Comment:
ich würde die node unter dem iterator erstellen und in der schleife nur zuweisen